### PR TITLE
Add option to remove quotations marks from returned strings

### PR DESCRIPTION
**Pull request by shalstein**
Changed the API to take an options object instead of a bunch plain parameters.

### CODE REVIEW

**Comment by Scimonster** [comment]
Thanks for the pull request. I like the idea but there are a few changes i'd like made first. I'm commenting on the relevant lines. Basically, please keep it backwards compatible, and ES5 compatible (so no const, no destructuring).

File: gematriya.js [hunk 1]
```diff
@@ -60,7 +60,8 @@
 		letters[numbers[i]] = i;
 	}
 
-	function gematriya(num, limit) {
+	function gematriya(num, options = {limit: false, punctuate: true, order: false }) {
```
Comment (Scimonster) [inline gematriya.js]: Parameter defaults are an ES6 feature, please use ES5.

File: gematriya.js [hunk 2]
```diff
@@ -60,7 +60,8 @@
...
+		const {limit, punctuate, order} = options;
```
Comment (Scimonster) [inline gematriya.js]: ES6 destructuring

File: gematriya.js (hunk 1 above)
Comment (Scimonster) [reply gematriya.js]: If this could work in a backwards-compatible manner that would be best.

**Review by Scimonster — CHANGES_REQUESTED** [review]

**Comment by shalstein** [comment]
I made those changes.

**Review by Scimonster — APPROVED** [review]

**Comment by Scimonster** [comment]
Thanks!

**Pull request merged**